sdk/keyvault/Azure.Security.KeyVault.Keys/src/Cryptography/Base/AlgorithmEnums.cs

```diff
@@ -0,0 +1,114 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Azure.Security.KeyVault.Cryptography.Base
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    internal enum AlgorithmCategoryEnum
```

Comment:
Not sure why this is necissary

Reply:
I had a reason for this in my earlier design, but now I can't even remember, so seems like a right time to delete it